spec/features/user_creates_new_book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User creates a new book' do
+  scenario 'successfully' do
+    visit root_path
+
+    create_book("The Lord of the Rings: The Fellowship of the Ring", "J. R. R. Tolkien" )
+
+    expect(page).to have_css('.books li',
+                    text: 'The Lord of the Rings: The Fellowship of the Ring')
```

Comment:
I would consider shortening the book title so this can all fit on one line. Maybe just "The Fellowship of the Ring"?